.github/workflows/dast-release.yml

```
@@ -0,0 +1,310 @@
+      contents: read
+      id-token: write
+      attestations: write
+    env:
+      ZAP_PREDICATE_TYPE: "https://security.sigstore.dev/attestation/dast/zap/v1"
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - name: backend
+            image_repo: app-stayhealthy-backend
+            image_digest: ${{ inputs.image_digest_backend }}
```

actionlint (1.7.12)

[error] property "image_digest_backend" is not defined in object type {image-digest-backend: string; image-digest-frontend: string; image-digest-worker: string}

(expression)